platform/rct/hal/LED_stm32/led_device_stm32.c

```diff
@@ -55,9 +55,9 @@ static bool led_set_level(struct led_data *ld, const bool on)
 
         ld->level = on;
         if (on) {
-                GPIO_SetBits(GPIO_PORT, ld->mask);
-        } else {
                 GPIO_ResetBits(GPIO_PORT, ld->mask);
+        } else {
+                GPIO_SetBits(GPIO_PORT, ld->mask);
```

Comment:
Seems like we should just move to an open collector instead of leaving the push pull setup.  Not sure if it makes much (if any) difference internally to the STM chip.